src/Routes/Devices/DeviceTable.js

```diff
@@ -307,7 +307,7 @@ const DeviceTable = ({
     if (isLoading) return actions;
     if (!rowData?.rowInfo?.id) return actions;
 
-    if (handleAddDevicesToGroup && groupActionsEnabled) {
+    if (handleAddDevicesToGroup && !groupActionsEnabled) {
```

Comment:
maybe rename the variable groupActionsEnabled to hideGroupsActions

Reply:
Renamed, thanks for the suggestion